Details/core/control.lua

```diff
@@ -314,6 +316,16 @@
 			_table_wipe(_details.cache_healing_group)
 			_details:UpdateParserGears()
 			
+			if (not _details:CaptureGet("damaage")) then _details:CaptureSet(true, "damage", true) end
```

Comment:
```suggestion
			if (not _details:CaptureGet("damage")) then _details:CaptureSet(true, "damage", true) end
```

Reply:
fixed in https://github.com/ajseward/Details-Damage-Meter-for-3.3.5a/pull/1